src/components/Hero.vue

```diff
@@ -5,10 +5,17 @@ const contentStore = useContentStore();
 const { getConstant } = contentStore;
 
 const trailer_embed = 'https://www.youtube.com/embed/Bxi-4XEANbE?autoplay=1&mute=1&loop=1&controls=0&modestbranding=1&playsinline=1&rel=0&enablejsapi=1&playlist=Bxi-4XEANbE&end=36';
+
+const scrollToBottom = () => {
+  window.scrollTo({
+    top: document.getElementById('hero')?.clientHeight,
```

Comment:
This doesn't take into account the element above and leave a small part of the hero one the screen. The solution is to add the y coordinate to the client height.